src/common/components/NavigationDrawer.vue

```diff
@@ -195,14 +217,17 @@ export default {
   },
 
   data: () => ({
-    bookIcon
+    bookIcon,
+    alertIcon,
+
+    version: ""
   }),
 
-  methods: {
-    openHref(href){
-      window.open(href, "_blank").focus()
-    },
+  created() {
+    this.version = `v${process.env.VUE_APP_VERSION}`
```

Comment:
Don't you think it's better to get the version from package.json to minimalize maintain version in 2 place? And also can you help change the version in package.json to 1.0.4?